### PR TITLE
[improve][doc] improve apache devlake website documentation

### PR DESCRIPTION
**Pull request by SnowMoon-Dev**
## Brief change log

1. uppdate devlake website description
2. update devlake website Developer Setup  chapter description

## Verify this pull request

Manually verified the change by testing locally

### CODE REVIEW

**Comment by warren830** [comment]
Thank you very much for your contribution! Can we exchange email info? We'd like to issue a certificate to you

**Comment by SnowMoon-Dev** [comment]
> Thank you very much for your contribution! Can we exchange email info? We'd like to issue a certificate to you

ok thanks, my email is shiminan95@gmail.com

File: docs/03-Plugins/refdiff.md [hunk 1]
```diff
@@ -84,8 +84,12 @@ make
 make install
 ```
 
-### MacOs
-
+### MacOS
+1. HomeBrew install
```
Comment (warren830) [inline docs/03-Plugins/refdiff.md]: HomeBrew will install libgit2 1.4.3, I think macport is better?

**Review by warren830 — CHANGES_REQUESTED** [review]

File: docs/03-Plugins/refdiff.md (hunk 1 above)
Comment (SnowMoon-Dev) [reply docs/03-Plugins/refdiff.md]: Yes, macports is better, it allows better version control

**Review by warren830 — APPROVED** [review]
LGTM

**Pull request merged**